codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/protocols/awsjson/AWSJSONCustomizations.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.aws.swift.codegen.protocols.awsjson
+
+import software.amazon.smithy.aws.swift.codegen.AWSClientRuntimeTypes
+import software.amazon.smithy.aws.swift.codegen.AWSHTTPProtocolCustomizations
+import software.amazon.smithy.codegen.core.Symbol
+import software.amazon.smithy.model.traits.TimestampFormatTrait
+
+class AWSJSONCustomizations : AWSHTTPProtocolCustomizations() {
+    override val baseErrorSymbol: Symbol = AWSClientRuntimeTypes.AWSJSON.AWSJSONError
+
+    override val defaultTimestampFormat = TimestampFormatTrait.Format.EPOCH_SECONDS
+}
```

Comment:
This type provides some protocol-specific values that used to be provided in the protocol generator.

Ideally, we would extract all the custom logic to a type like this and not need protocol generators for every protocol.